src/providers/fault-count/cat-d/fault-count.cat-d.ts

```diff
@@ -193,9 +205,9 @@ export class FaultCountDHelper {
     const result =
       faultTotal +
       sumManoeuvreFaults(manoeuvres, CompetencyOutcome.DF) +
-      FaultCountDHelper.getVehicleChecksFaultCountCatD(vehicleChecks).drivingFaults +
-      pcvDoorExerciseFaultCount +
-      uncoupleRecoupleHasDrivingFault;
+      FaultCountDHelper.getVehicleChecksFaultCountTrailer(vehicleChecks).drivingFaults +
```

Comment:
should be:

`this.getVehicleChecksFaultCountTrailer(vehicleChecks).drivingFaults`